### PR TITLE
165364032-fixing seeding dates

### PR DESCRIPTION
**Pull request by ljmars2351**
fixing the UI side of the date problem as well as updating the version

### CODE REVIEW

File: projects/revaturexyz/forecast/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "license": "MIT",
   "name": "@revaturexyz/forecast",
-  "version": "0.3.6",
+  "version": "0.3.8",
```
Comment (stormrubio3000) [inline projects/revaturexyz/forecast/package.json]: is this supposed to be 8 or 7?

**Comment by escalonn** [comment]
must be up to date with broker.

File: projects/revaturexyz/forecast/package.json (hunk 1 above)
Comment (ljmars2351) [reply projects/revaturexyz/forecast/package.json]: 8, 7 is current

Comment (ljmars2351) [reply projects/revaturexyz/forecast/package.json]: at least thats what Nick said

**Comment by escalonn** [comment]
probably a bad merge?

**Pull request closed without merging**